<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_main">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>731</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Gestion des vehicules</string>
      </property>
      <widget class="QPushButton" name="pushButton_ajou">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Ajouter vehicule</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_aff">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>140</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Afficher vehicule</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_back">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>251</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_ajout">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>631</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter vehicule</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de vehicule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Marque:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Puissance:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Energie:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de mise en service:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_type">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_marque">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_matricule">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_puissance">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_energie">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_aj1">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>250</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_bkaj">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>270</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(144, 134, 120);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_date">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_aff">
     <widget class="QPushButton" name="pushButton_bkaf">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(144, 134, 120);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>20</y>
        <width>191</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(233,248,255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QRadioButton" name="radioButton_mat">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
padding: 5px;
border-radius: 5px;

font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_date">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
padding: 5px;
border-radius: 5px;

font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_73">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
        </font>
       </property>
       <property name="text">
        <string>Trier selon :</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_71">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Matricule:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_rech">
      <property name="geometry">
       <rect>
        <x>624</x>
        <y>160</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_imp">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>280</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
      </property>
      <property name="text">
       <string>Imprimer</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>251</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(233,248,255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
      <widget class="QPushButton" name="pushButton_modi">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type de vehicule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Marque:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Puissance:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Energie:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de mise en service:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>160</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tabView_vehicule">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>21</y>
        <width>441</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollBar:vertical {
                 background: #0094ff;
                }
                QScrollBar:horizontal {
                 background: #0094ff;
                }

QTableView  
{  
  qproperty-currentItemBackground: #cccccc;  
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_surema">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>130</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supp">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 5px;
background-color:rgb(101, 190, 155);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
